packages/lesswrong/components/search/PostsListEditorSearchHit.tsx

```diff
@@ -47,7 +47,7 @@ const PostsListEditorSearchHit = ({hit, classes}: {
         <PostsPreviewTooltipSingle postId={post._id} postsList/>
       </LWPopper>
       <div>
-        <PostsTitle post={post as unknown as PostsBase} isLink={false}/>
+        <PostsTitle post={post as unknown as PostsListBase} isLink={false}/>
```

Comment:
The `post` prop of `PostsTitle` previously had type `PostsBase`, which doesn't have a `tags` property. The expanded `PostsListBase` type does have access to tags, which I used to determine which icon to use. Every other use of the `PostsTitle` component passes in a `post` of type `PostsListBase` (or an extension thereof) except this one, and this case is just a workaround to avoid ts errors from `AlgoliaPost`. (The commit message for this line's last edit is "Awkwardly pretend AlgoliaPost isn't missing fields PostsTitle could use (but doesn't strictly need)".)

Definitely something I will notify the LW team about.